optaplanner-docs/src/main/asciidoc/Chapter-Quick_start/Chapter-Quick_start.adoc

```diff
@@ -445,14 +467,18 @@ public class CloudBalancingEasyScoreCalculator implements EasyScoreCalculator<Cl
 ----
 ====
 
-Even if we optimize the code above to use ``Map``s to iterate through the `processList` only once, *it is still slow* because it does not do <<incrementalScoreCalculation,incremental score calculation>>.
+Even if we optimize the code above to use ``Map``s to iterate through the `processList` only once, it is still slow because it does not do <<incrementalScoreCalculation,incremental score calculation>>.
```

Comment:
Normally we wouldn't emphasise something in the middle of a sentence like this. Just from a stylistic point of view, it seems a little casual for a technical document. However, if you really want to emphasise it, the IBM Style Guide recommends italics.